src/models/generated/StandardLoad.jl

```diff
@@ -28,25 +28,29 @@ This file is auto-generated. Do not edit.
         internal::InfrastructureSystemsInternal
     end
 
-Data structure for a standard load.
+A voltage-dependent [ZIP load](@ref Z), most commonly used for dynamics modeling.
+
+A `StandardLoad` breaks the ZIP into three pieces: Z (constant impedance), I (constant current), and P (constant power), according to `P = P_P * V^0 + P_I * V^1 + P_Z * V^2` for active power and `Q = Q_P * V^0 + Q_I * V^1 + Q_Z * V^2` for reactive power. (Voltage V is in per unit.)
+
+For an alternative exponential formulation of the ZIP model, see [`ExponentialLoad`](@ref). For a simpler load model with no voltage dependency, see [`PowerLoad`](@ref).
 
 # Arguments
 - `name::String`: Name of the component. Components of the same type (e.g., `PowerLoad`) must have unique names, but components of different types (e.g., `PowerLoad` and `ACBus`) can have the same name.
 - `available::Bool`: Indicator of whether the component is connected and online (`true`) or disconnected, offline, or down (`false`). Unavailable components are excluded during simulations.
 - `bus::ACBus`: Bus that this component is connected to
 - `base_power::Float64`: Base power of the load (MVA) for per unitization., validation range: `(0, nothing)`, action if invalid: `warn`
-- `constant_active_power::Float64`: (optional)
-- `constant_reactive_power::Float64`: (optional)
-- `impedance_active_power::Float64`: (optional)
-- `impedance_reactive_power::Float64`: (optional)
-- `current_active_power::Float64`: (optional)
-- `current_reactive_power::Float64`: (optional)
-- `max_constant_active_power::Float64`
-- `max_constant_reactive_power::Float64`: (optional)
-- `max_impedance_active_power::Float64`: (optional)
-- `max_impedance_reactive_power::Float64`: (optional)
-- `max_current_active_power::Float64`: (optional)
-- `max_current_reactive_power::Float64`: (optional)
+- `constant_active_power::Float64`: (optional) Constant active power demand in MW (P_P)
```

Comment:
None of this are optional. The value 0.0 is a valid quantity

Reply:
Optional isn't meant to mean that 0.0 isn't valid. We're using "optional" for everything that has a valid default value, but the user could change it to some other valid value if they want. It's just not something you have to define, because you could leave it at 0.0.

If there's no default, it's not optional and you have to define it. 

If it has a default value, but it really shouldn't be changed (like an expression calculated from other input parameters), we've labeled it "Do not modify."

Reply:
`Optional` can be a confusing term. In some programming languagues `Optional` in the context of a type means that a value can be null. Julia doesn't have that, but we use the concept in this package.

We could sidestep this question by auto-generating a string with the default value. For example, this string could be generated as
```
- `constant_active_power::Float64`: (default=0.0) Constant active power demand in MW (P_P)
```

Reply:
Jose and I had also talked about eventually auto-generating the default value. If we can do that now, I think that would be the best solution, and then we can just delete all the optionals.

Reply:
@daniel-thom  It's a change in the template in IS we can do to always place the default.

Reply:
Rodrigo and I added a print out of the default value in IS. https://github.com/NREL-Sienna/InfrastructureSystems.jl/pull/374 

Example for StandardLoad:
![Screenshot 2024-06-04 at 2 45 14 PM](https://github.com/NREL-Sienna/PowerSystems.jl/assets/12451220/9eab35a9-b689-4f79-b39a-05d58bdc8f8d)

Reply:
Removed manual optional in https://github.com/NREL-Sienna/PowerSystems.jl/pull/1111/commits/a23061991537da8536d887fed5ae169bdc76e05b